packages/zowe-explorer/src/configuration/Profiles.ts

```diff
@@ -530,15 +530,15 @@ export class Profiles extends ProfilesCache {
             switch (choice) {
                 case "project":
                     for (const file of existingLayers) {
-                        if (file.user) {
+                        if (!file.path.includes(FileManagement.getZoweDir())) {
```

Comment:
I believe that the only scenario that this won't work is if someone opens their `ZOWE_CLI_HOME` folder and wants to edit the configuration  and select project config.
Since the file.path will in fact be the same as the getZoweDir(), it won't trigger the openConfigFile 😅 

That is a rare enough scenario though.